### ISSUE TITLE
Fix location pagination links

### ISSUE THREAD

**Issue by PhantomWatson** [BallStateCBER/muncie-events#47]
- [This location page](https://muncieevents.com/location/Minnetrista) has a "next" pagination link to `https://muncieevents.com/location/Minnetrista/page:2`,
 - Page 2 shows past events, even though it should only be showing future events
 - Also, `page:2` appears to not be read correctly by the pagination component, as that page has a "next" pagination link to `https://muncieevents.com/location/Minnetrista/page%3A2/page:2`
 - And shows upcoming events
 - And that page has a "next" link back to `https://muncieevents.com/location/Minnetrista/page:2`

So basically, the "next" button just loops the user through the first "upcoming" page and the first "all/past" page.
